doc/sphinx/addendum/extraction.rst

```diff
@@ -47,30 +47,30 @@ extraction. They both display extracted term(s) inside |Coq|.
 All the following commands produce real ML files. User can choose to
 produce one monolithic file or one file per |Coq| library.
 
-.. cmd:: Extraction "@file" {+ @qualid }
+.. cmd:: Extraction @string {+ @qualid }
 
    Recursive extraction of all the mentioned objects and all
-   their dependencies in one monolithic `file`.
+   their dependencies in one monolithic file :token:`string`.
```

Comment:
Hmm, `@file` was nice for documentation purposes though

Reply:
I agree but this one the only use of this token in the all manual. Not enough to define a new grammar token.